File: tvguide-api/src/domain/channel.ts
export interface Channel {
  id: string;            // stabiele eigen id, bv "npo1"
  sourceId: string;      // tvgids ch_id, bv "1"
  name: string;
  logoUrl: string | null;
  inNlziet: boolean;     // stuurt de zenderselectie
  nlzietSlug: string | null;
  sortOrder: number;
}

File: android/app/src/main/java/com/nexustvguide/app/core/domain/Channel.kt
package com.nexustvguide.app.core.domain

data class Channel(
    val id: String,
    val sourceId: String,
    val name: String,
    val logoUrl: String? = null,
    val inNlziet: Boolean = false,
    val nlzietSlug: String? = null,
    val nlzietChannelId: String? = null,
    val sortOrder: Int = 999
)
